DQM/Physics/python/topSingleLeptonDQM_cfi.py

```diff
@@ -317,12 +55,13 @@
       select     = cms.string(tightEleCut + "& pt>20 & abs(eta)<2.5 & (abs(gsfElectronRef.superCluster().eta()) <= 1.4442 || abs(gsfElectronRef.superCluster().eta()) >= 1.5660)"),
 			rho = cms.InputTag("fixedGridRhoFastjetAll"),
     ),                                     
-    muonExtras = cms.PSet(                                               
-      select    = cms.string(looseMuonCut + " && pt>20 & abs(eta)<2.1"),                                              
+    muonExtras = cms.PSet(
+      select    = cms.string(tightMuonCut + " && pt>20 & abs(eta)<2.4"),                                               
+      #select    = cms.string(looseMuonCut + " && pt>20 & abs(eta)<2.4"),                                              
       isolation = cms.string(looseIsoCut)
     ),
     jetExtras = cms.PSet(
-      jetCorrector = cms.InputTag("dqmAk4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!                                            
+      jetCorrector = cms.InputTag("ak4PFCHSL1FastL2L3Corrector"),  #Use pak4PFCHSL1FastL2L3Residual for data!!!                                            
```

Comment:
This is breaking again the issue found in https://github.com/cms-sw/cmssw/issues/22996
and fixed in https://github.com/cms-sw/cmssw/pull/28156
Please revert the change and test it getting the Tier0 configuration as explained in https://its.cern.ch/jira/browse/CMSTZ-427
Thanks